contracts/BaseStrategy.sol

```diff
@@ -317,16 +310,14 @@ abstract contract BaseStrategy {
         outstanding = vault.report(want.balanceOf(address(this)).sub(reserve));
     }
 
-    // Override this to add all tokens this contract manages on a *persistant* basis
-    // (e.g. not just for swapping back to want ephemerally)
-    // NOTE: Must inclide `want` token
-    function protectedTokens() internal virtual view returns (address[] memory) {
-        address[] memory protected = new address[](1);
-        protected[0] = address(want);
-        return protected;
-    }
+    // Override this to add all tokens/tokenized positions this contract manages
+    // on a *persistant* basis (e.g. not just for swapping back to want ephemerally)
+    // NOTE: Do *not* include `want`, already included in `sweep` below
+    function protectedTokens() internal virtual view returns (address[] memory);
 
     function sweep(address _token) external {
```

Comment:
I don't think there is any need for this to be open to public. I can only think of a few very fringe situations where governance wouldn't want to do it themselves but someone else would. And there is a good chance someone will mess up the protectedTokens when writing a strategy at some point in the future.

Reply:
Solved in 19e0ca5